packages/wallet/ui/src/contexts/Application.jsx

```diff
@@ -9,8 +11,38 @@ import {
   useReducer,
 } from 'react';
 
+// TODO: Graduate to show mainnet in production.
+const SHOW_MAINNET = process.env.NODE_ENV === 'development';
+export const DEFAULT_WALLET_CONNECTIONS = [
```

Comment:
I'm impressed at the non-imperative here, but it is pretty hard to read. A quick fix would be to conditionally push onto the array. Another would be to name values then filter out empties,
```
const localConnection = window && window.location.hostname !== 'localhost'
  ? { label: window.location.hostname, url: window.location.origin }
  : { label: 'Local Solo', url: 'http://localhost:8000' }]
const stageConnection = process.env.NODE_ENV === 'development' && {
          label: 'Agoric Stage',
          url: 'https://stage.agoric.net/network-config',
        };
  …
export const DEFAULT_WALLET_CONNECTIONS = [
  LOCAL_CONNECTION,
  NET_CONNECTION,
  STAGE_CONNECTION,
  ].filter(Boolean);
```